.github/workflows/ci.yml

```diff
@@ -111,6 +111,76 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
 
+  build-standalone:
```

Comment:
There are lots of steps in here that create artifacts which aren't needed for the build.
Is the name still precise?